cmd/swarm/config.go

```diff
@@ -288,9 +284,9 @@ func envVarsOverride(currentConfig *bzzapi.Config) (config *bzzapi.Config) {
 		}
 	}
 
-	if syncenable := os.Getenv(SWARM_ENV_SYNC_ENABLE); syncenable != "" {
-		if sync, err := strconv.ParseBool(syncenable); err != nil {
-			currentConfig.SyncEnabled = sync
+	if syncdisable := os.Getenv(SWARM_ENV_SYNC_DISABLE); syncdisable != "" {
+		if sync, err := strconv.ParseBool(syncdisable); err != nil {
```

Comment:
I know it's paranoid, but should we check `sync` to actually be set to `true` before this? like `err != nil && sync {`

Reply:
If the env var is set to `false` the value of `api.Config.SwarmEnabled` will be `true`. This is correct, no?